From foo@baz Sun Jan 25 10:05:08 PST 2009
Date: Sun, 25 Jan 2009 10:05:08 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: staging: dst: replace bus_id with dev_set_name

bus_id is going away, use the dev_set_name() function instead.

Cc: Kay Sievers <kay.sievers@vrfy.org>
Cc: Evgeniy Polyakov <zbr@ioremap.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/dst/dcore.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/dst/dcore.c
+++ b/drivers/staging/dst/dcore.c
@@ -311,7 +311,7 @@ static int dst_node_sysfs_init(struct ds
 	memcpy(&n->info->device, &dst_node_dev, sizeof(struct device));
 	n->info->size = n->size;
 
-	snprintf(n->info->device.bus_id, sizeof(n->info->device.bus_id), "dst-%s", n->name);
+	dev_set_name(&n->info->device, "dst-%s", n->name);
 	err = device_register(&n->info->device);
 	if (err) {
 		dprintk(KERN_ERR "Failed to register node '%s', err: %d.\n",
